build.psm1

```diff
@@ -196,7 +196,7 @@ function Start-PSBuild {
         Write-Warning @"
 The currently installed .NET Command Line Tools is not the required version.
 
-Installed version: $dotnetCLIIntalledVersion 
+Installed version: $dotnetCLIIntalledVersion
```

Comment:
That wasn't part of this PR, but since I only see 3 hits, I'm leaning towards fixing it here instead of a new PR